medical/models/medical_physician.py

```diff
@@ -111,102 +107,3 @@ def create(self, cr, uid, vals, context=None):
         vals['groups_id'] = [(6, 0, group_ids)]
```

Comment:
Does anyone know the intent behind this `groups_ids`? It doesn't exist in the parent model `res.partner`, and isn't declared here.

I see it removed from `medical.patient`, just wanted to confirm there wasn't an awesome intent/idea behind this that should be preserved.